### PR TITLE
Feature/db

### PR DESCRIPTION
**Pull request by dulynoded**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const insertUser = `CREATE (n:User
+  {
+  customer_id: '${shortid.generate()}',
+  star_pref: ${Math.random()},
+  distance_pref: ${Math.random()},
+  price_pref: ${Math.random()},
+  openness: ${Math.random()},
+  hometown_latitude: ${faker.address.latitude()},
+  hometown_longitude: ${faker.address.longitude()},
+  personality: '[${Math.random()}, ${Math.random()}, ${Math.random()}, ${Math.random()}, ${Math.random()}]',
```
Comment (cjvil) [inline database/index.js:18]: any way to avoid raw query here?

File: example_data/users.js [hunk 2]
```diff
@@ -0,0 +1,848 @@
+const faker = require('faker');
+const fs = require('fs');
+const generateUsers = (num) => {
+  let test_users = []
+  for (var i = 0; i < num; i++) {
+    test_users.push(
+      {
+        ustomer_id: Math.floor(Math.random()*10000000),
```
Comment (cjvil) [inline example_data/users.js:8]: typo alert?

File: example_data/users.js [hunk 3]
```diff
@@ -0,0 +1,848 @@
...
+        star_pref: Math.random(),
+        distance_pref: Math.random(),
+        price_pref: Math.random(),
+        openness: Math.random(),
+        hometown_latitude: faker.address.latitude(),
+        hometown_longitude: faker.address.longitude(),
+        "personality": [
```
Comment (cjvil) [inline example_data/users.js:15]: doesn't look like new contract with customer profiling, assuming this is going to be revised?

File: example_data/users.js (hunk 2 above)
Comment (dulynoded) [reply example_data/users.js:8]: oh are you calling them customers?

**Pull request closed without merging**